src/main/resources/swagger.yaml

```diff
@@ -0,0 +1,949 @@
+          in: query
+          required: true
+          #description: Value true indicates that response to be jwt
+          schema:
+            type : string        
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                title: IntrospectionResponse
+                $ref: '#/components/schemas/IntrospectionResponse'                  
+        400:
+          description: Invalid parameters are provided to endpoint.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'        
+        500:
+          description: Introspection Internal Server Failed.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'   
+  /introspection(post):
+    post:
+      tags:
+      - developers
+      summary: The Introspection OAuth 2 Endpoint responds to HTTP POST requests from token holders. 
+      description: The Introspection Endpoint takes a single parameter representing the token and returns a JSON document representing the token meta information.
+      operationId: introspectPost      
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              required:
+              - token
+              - token_type_hint
+              - response_as_jwt
+              properties:
+                token:
+                  type : string
+                  description: Client access token.
+                token_type_hint:
+                  type : string  
+                  description: ID Token previously issued by the Authorization Server being passed as a hint about the End-User.
+                response_as_jwt:
+                  type : string
+                  description: Value true indicates that response to be jwt
+      parameters:
+        - name: Authorization
+          in: header
+          required: true
+          description: Client Authorization details that contains the access token along with other details.  
+          schema:
+            type : string 
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                title: IntrospectionResponse
+                $ref: '#/components/schemas/IntrospectionResponse'                                        
+        400:
+          description: Invalid parameters are provided to endpoint.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'        
+        500:
+          description: Introspection Internal Server Failed.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'  
+  /.jwks:
```

Comment:
Do we really have `.` (dot) before `jwks` ?

Reply:
My bad, have rectified this